posthog/celery.py

```diff
@@ -20,6 +20,10 @@
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks()
 
+# Make sure Redis doesn't add too many connections
+# https://stackoverflow.com/questions/47106592/redis-connections-not-being-released-after-celery-task-is-complete
+app.conf.broker_pool_limit = 0
```

Comment:
makes sense. curious if needing to open and close connections will become a bottleneck anyway but the load testing seems to hold up

Reply:
Hm yes I think it does slow it down a little bit but at least it works, especially with the cheaper Redis instances